xc/xenops_server_xen.ml

```diff
@@ -1581,6 +1587,10 @@ module VM = struct
 									(fun fd ->
 										Domain.restore task ~xc ~xs ~store_domid ~console_domid ~no_incr_generationid (* XXX progress_callback *) ~timeoffset ~extras build_info (choose_xenguest vm.Vm.platformdata) domid fd
 									);
+								try
```

Comment:
As I said, please outside the outer try/with block, and all the way at the end, so after L1600.